parse_ply.c

```diff
@@ -0,0 +1,144 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   parse_ply.c                                        :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: tyamagis <tyamagis@student.42tokyo.>       +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2022/02/16 14:56:58 by tyamagis          #+#    #+#             */
+/*   Updated: 2022/02/16 18:26:32 by tyamagis         ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "./include/ply.h"
+
+void	*poly_to_tri(t_ply *ply, int n, FILE **f_stream)
```

Comment:
"ply" is too abstract for me.
"ply_info" is more stress free for me.

Reply:
DONE